en/docs/reference/product-apis/service-catalog-apis/service-catalog-v1/service-catalog-v1.yaml

```diff
@@ -0,0 +1,1089 @@
+      tags:
+        - Services
+      parameters:
+        - name: overwrite
+          in: query
+          description: |
+            Whether to overwrite if there is any existing service with the same name and version.
+          required: false
+          schema:
+            type: boolean
+            default: false
+      requestBody:
+        required: false
+        content:
+          multipart/form-data:
+            schema:
+              required:
+                - file
+              properties:
+                file:
+                  type: string
+                  description: |
+                    Zip archive consisting of exported Application Configuration.
+                  format: binary
+                verifier:
+                  type: string
+      responses:
+        200:
+          description: |
+            Successful response with the imported service metadata.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceInfoList'
+          headers:
+            ETag:
+              $ref: '#/components/headers/ETag'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        500:
+          $ref: '#/components/responses/InternalServerError'
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+              -F "file=@services.zip" "https://127.0.0.1:9443/api/am/service-catalog/v0/services/import?overwrite=true"'
+                
+  #----------------------------------------------------------
+  # Exporting a service as a zipped archive
+  #----------------------------------------------------------
+  /services/export:
+
+    #-----------------------------------------
+    # Retrieve a service as a zipped archive
+    #-----------------------------------------
+    get:
+      summary: Export a service
+      description: |
+        Export a service as an archived zip file.
+      security:
+        - OAuth2Security:
+            - service_catalog:service_view
+      operationId: exportService
+      tags:
+        - Services
+      parameters:
+        - name: name
+          in: query
+          description: |
+            Name of the service to export
+          required: true
+          schema:
+            type: string
+        - name: version
+          in: query
+          description: |
+            Version of the service to export
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: |
+            Successful response as the exported service as a zipped archive.
+          content:
+            application/zip:
+              schema:
+                type: string
+                format: binary
+          headers:
+            ETag:
+              $ref: '#/components/headers/ETag'
```

Comment:
Let's remove the unsupported header.

Reply:
Removed in 0e2d05ca1eb804b7d1352c77d474e1c01accdbdd